src/main/java/org/springframework/integration/aws/config/xml/AmazonWSParserUtils.java

```diff
@@ -48,11 +57,10 @@ private AmazonWSParserUtils() {
 	 * Registers the {@link AWSCredentials} bean with the current ApplicationContext if
 	 * accessKey and secretKey is given, if the credentials-ref is given, the given value
 	 * is returned.
-	 *
 	 * @param element
 	 * @param parserContext
 	 */
-	public static String getAmazonWSCredentials(Element element,ParserContext parserContext) {
+	public static String getAmazonWSCredentials(Element element, ParserContext parserContext) {
```

Comment:
@aemruli , thank you very much for the feedback!
Please, don't pay attention to this Credentials stuff yet.
We really are going to get rid of that and rely on the your foundation.
Right now our goal to introduce as much new features as possible: SQS, SNS, Kinesis.
After that we'll come back to this old solution and override it right way, as well as the current S3 solution.